.dockerignore

```
@@ -2,7 +2,6 @@
 /.git
```

すごく細かいんだけれどついでに`/.circleci`も追加されている方がいいと思った！
```suggestion
/.circleci
/.git
```

---

お、確かにそうだね！コメントありがとう🐱！